src/app/data-set/data-set.component.html

```diff
@@ -0,0 +1,33 @@
+<button mat-raised-button (click)="addDataSets()" color="warn">Add projects</button>
+
+<div class="example-container mat-elevation-z8">
+    <mat-table #table [dataSource]="dataSource">
+        <ng-container matColumnDef="id">
```

Comment:
IDs are something machines care for and should not be displayed to humans. When we add the name we can display it here.